storagedriver/azure/azure.go

```diff
@@ -174,11 +193,17 @@ func (d *Driver) WriteStream(path string, offset, size int64, reader io.ReadClos
 		}
 	}
 
-	if offset != resumableOffset {
-		return storagedriver.InvalidOffsetError{Path: path, Offset: offset}
+	if offset < resumableOffset {
```

Comment:
Hmm, as long as the path remains the same, the subsequent writes (even with offset=0) will also fail. we should support this definitely, if needed. Do you mind writing the test case and let azure fail on that while I fix the driver on a separate PR?